Wake On Lan/Modules/AboutScreen/Sources/AboutScreen/AboutScreenHeaderViewViewModel.swift

```diff
@@ -4,8 +4,8 @@
 //
 //  Created by Dmitry on 08.08.2021.
 //
-
-struct AboutScreenHeaderViewViewModel {
+// sourcery: AutoEquatable
+public struct AboutScreenHeaderViewViewModel {
```

Comment:
This struct should not be public

Reply:
When I have this struct internal, I get an error in AutoEquatable file:  Method '==' must be declared public because it matches a requirement in public protocol 'Equatable'.
Did I make a mistake in a sourcery file?
If the struct must remain internal I can also just check both parameters of the viewModel or make it conform to equatable without using sourcery.

Reply:
update: as discussed, I created a pull request for Swift-Templates submodule to make AutoEquatable == method always public. Now I can make the struct internal again.